commitlint/plugins.test.ts

```diff
@@ -652,6 +652,30 @@ test("proper-issue-refs3", () => {
     expect(properIssueRefs3.status).toBe(0);
 });
 
+test("reject-obvious-words1", () => {
+    let commitMsgWithObviousWordAfterColon = "foo: modify bla bla";
+    let rejectObviousWord1 = runCommitLintOnMsg(
+        commitMsgWithObviousWordAfterColon
+    );
+    expect(rejectObviousWord1.status).not.toBe(0);
+});
+
+test("reject-obvious-words2", () => {
+    let commitMsgWithObviousWordAfterColon = "foo: change bla bla";
+    let rejectObviousWord2 = runCommitLintOnMsg(
+        commitMsgWithObviousWordAfterColon
+    );
+    expect(rejectObviousWord2.status).not.toBe(0);
+});
+
+test("reject-obvious-words3", () => {
+    let commitMsgWithoutObviousWordAfterColon = "foo: bla bla bla";
+    let rejectObviousWord3 = runCommitLintOnMsg(
+        commitMsgWithoutObviousWordAfterColon
+    );
+    expect(rejectObviousWord3.status).toBe(0);
+});
```

Comment:
@realmarv I've slightly changed my mind of how this rule should work, but instead of explaining it, I'll just request you to add the following tests:
- `foo: update bla bla` -> status should not be 0
- `foo: update` -> status should be 0
- `foo: change` -> status should not be 0
- `foo: update bla bla\n\nBla blah.` -> status should be 0
- `foo: change bla bla\n\nBla blah.` -> status should not be 0

That is, there is an exception to the rule: the word update can go alone after area ("change" or "modify" cannot). The word `update` can only go with extra words after it if body length is higher than zero ("change" or "modify" cannot).

The above means that you should not make this rule extensible (i.e. by putting the list of words in a separate file or list). You have to hardcode them in the plugin instead.

Reply:
Done